tests/templates/test_subroutines/test_select.py

```diff
@@ -19,19 +19,22 @@
 
 import numpy as np
 import pytest
+from scipy.stats import unitary_group
 
 import pennylane as qml
 from pennylane import numpy as pnp
+from pennylane.templates.subroutines.select import _unary_select
 
 
 def test_standard_checks():
     """Run standard validity tests."""
-    ops = [qml.PauliX(0), qml.PauliY(0)]
-    control = [1]
+    ops = [qml.PauliX(0) for _ in range(16)]
```

Comment:
If instead of 16 we write 15, the test fails. The reason is [this one](https://xanaduhq.slack.com/archives/C06VCNYC3LH/p1748955221402189)
We could add a note maybe?

Reply:
([I asked here](https://xanaduhq.slack.com/archives/C0271MF2M50/p1749666531186959))

Reply:
After this is solved, we should be able to pass the codecov test by adding non-power of 2 examples